jscripts/normalizeCharacters.js

```diff
@@ -0,0 +1,30 @@
+function normalizeCharacters()
+	{
+		// Find searcArg and replace smart quote with single quote 
+		if (document.getElementById("searchArg")){	
```

Comment:
@aprigge This works correctly now, via all search boxes.  Great!

Now, let's improve this by reducing the repeated code.  All 4 of these are essentially the same: they take an input element's name, and if it exists, they update the element's value.

I recommend unifying this - could be a separate function, or just an inline block of code.  Maybe iterate over a list of element names, using that to call the same code 4 times, rather than 4 blocks of code like this.

This would also make it easier to expand the normalization to other characters, so those changes would be in one place instead of multiple - but that will be a possible separate Jira issue.

Thanks --Andy

Reply:
Thanks, @akohler. I've updated the function to use iteration to apply the normalization change. Let me know what you think.